[IMP] Pos automatic solve module

quentinDupont opened the pull request:
It allows to quickly solve multiple control differences in point of sale with a simple button.
Used cash.box.out/in methods.

Parameters in POS config (see configuration part) defined the absolute limit when we can use the automatic solve button.

See README for images

Commit:
[IMP] Quick automatic solve for multiple control

legalsylvain commented on pos_multiple_control/models/account_bank_statement.py:
```diff
@@ -24,6 +26,14 @@ class AccountBankStatement(models.Model):
 
+    pos_session_state = fields.Char(
+        string='Pos session state',
+        compute='_compute_pos_session_state')
```
don't remember, but fied;related doesn't work ? 


    pos_session_state = fields.Char(
        related='pos_session_id.state')

legalsylvain commented on pos_multiple_control/i18n/fr.po:
```diff
@@ -60,6 +72,11 @@ msgstr "Balance de fermeture"
+#. module: pos_multiple_control
+#: view:pos.session:pos_multiple_control.view_pos_session_form
+msgid "Do you really want to solve this difference automatically ?"
+msgstr "Veux-tu vraiment solder cette différence automatiquement ?"
```
Tu / vous dans les traductions.

legalsylvain reviewed (approved):
2 minors remarks. LGTM otherwise ! 
First tests ! nice !

legalsylvain committed:
[FIX] accounting problem during closing session (+flake8)

Use Case : close the session, the account move line generated is doesn't have the account defined in the product.
